libdnf/hy-package.cpp

```diff
@@ -943,6 +945,39 @@ dnf_package_get_files(DnfPackage *pkg)
     return (gchar**)g_ptr_array_free (ret, FALSE);
 }
 
+/**
+ * dnf_package_get_changelogs:
+ * @pkg: a #DnfPackage instance.
+ *
+ * Gets the list of changelogs for the package.
+ *
+ * Returns: (transfer container) (element-type Changelog): a list
+ *
+ * Since: 0.19.1
+ */
+std::vector<std::unique_ptr<libdnf::Changelog>>
```

Comment:
What about using std::vector<libdnf::Changelog> instead of std::vector<std::unique_ptr<libdnf::Changelog>>?